### PR TITLE
Added some coins to the maint loot spawner

### PR DESCRIPTION
**Pull request by Jujumatic**
Added some coins to the maintenance loot spawner for botanists to hunt for to get them strange seed packets. this is a followup from #7374

### CODE REVIEW

**Comment by Jujumatic** [comment]
why did this throw in the diamond coins, ugh one sec

File: hippiestation/code/_globalvars/lists/maintenance_loot.dm [hunk 1]
```diff
@@ -106,5 +106,8 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/toy/eightball = 1,
 	/obj/item/stack/ducttape = 5,
 	/obj/item/clothing/under/hippie/cosby = 3,
+	/obj/item/coin/gold = 3,
+	/obj/item/coin/silver = 3,
```
Comment (Pyko1) [inline hippiestation/code/_globalvars/lists/maintenance_loot.dm]: there is already a silver coin on the line 62

Comment (Jujumatic) [reply hippiestation/code/_globalvars/lists/maintenance_loot.dm]: fixed

**Comment by praisenarsie** [comment]
or just get miners to make coins

**Comment by Jujumatic** [comment]
>expecting miners to do their job

LMAO

remember how we're supposed to try and get every department to be as independent as possible because hippie players don't help each other?

**Comment by JohnGinnane** [comment]
yeah but there's other ways to get coins, and I still think makes items being premium completely redundant

**Comment by Jujumatic** [comment]
the point isn't to make the items horrendously difficult to get - this makes sure that they have to put in *some* effort to get those coins, and finding just one coin doesn't mean they can get all the premium items from their seed vendor - unless it's a diamond coin, which will be difficult to find anyway?

**Comment by JohnGinnane** [comment]
There's a way of re-using a single coin to get premium items and it's very easy to do

**Comment by Jujumatic** [comment]
but that's literally an exploit isn't it

**Comment by JohnGinnane** [comment]
no it's actually a feature

image

**Comment by Jujumatic** [comment]
unless you're talking about the coin on a string

**Comment by Jujumatic** [comment]
i mean i don't see why this is a problem when there's coins lying around in a few places anyway, this just gives more options

**Comment by JohnGinnane** [comment]
but that's the problem, there's too many options and it defeats the purpose. The only location a botanist can immediately get a coin is dorms and a few people want that for things like ins gloves

My point is, when you give them a coin to access the premium content there's no reason for the content to be premium because it's no longer gated effectively

**Comment by Jujumatic** [comment]
so then, if they want coins, where exactly do they go? assuming they don't use the coin on a string

**Comment by Pyko1** [comment]
search for one in maint or rob vault?

**Comment by JohnGinnane** [comment]
the same places as everyone else, or maint, or vault, or cargo

**Comment by Jujumatic** [comment]
i wanted to add this as an optional minigame kinda thing, look for the coin. i still don't see how this makes the premium content redundant since this still takes work to do

**Comment by JohnGinnane** [comment]
Sorry, I didn't realise this just adds them to maint, my bad. In that case it's probably not as bad, as botanists don't start with maint access

Although I'm not sure of the effects this has elsewhere

**Comment by Jujumatic** [comment]
also

vault has one coin, that can be made into like 3 silver coins at most

**Comment by Jujumatic** [comment]
SHIT i accidentally closed this

**Comment by Pyko1** [comment]
because 1 diamond coin is prettymuch enough to empty every premium.

I mean coins could be more common, but id rather see the silver and cheaper coins buffed slighly.

**Comment by Jujumatic** [comment]
the 1 diamond coin is pretty rare though, that's why i put it to only 1 in the loot spawner. i could change it to 1 plasma coin instead if you think diamond is too much

**Comment by JohnGinnane** [comment]
There's at least one static spawn diamond coin, in maint bar above primary tool storage

**Comment by Pyko1** [comment]
id rather see something like
/obj/item/coin/twoheaded = 3
/obj/item/coin/silver = 2

and leave the expensive coins out, just couple more coins and the coin string is already plenty

**Comment by Jujumatic** [comment]
yeah but it's either the RD or botany fighting over for it, that's why i wanted to add another. by putting it in the loot spawner it means someone has to do some searching to find it, in which case they'll probably come across the other coins instead

**Comment by Pyko1** [comment]
why does it need to be diamond coin?

**Comment by Jujumatic** [comment]
if botany gets lucky and finds it, it means they don't have to go searching for more coins for the round in order to get their seeds and can instead just focus on growing all the strange seeds. if the RD finds it he can throw it into the RnD machine for the sweet sweet diamonds

**Comment by Jujumatic** [comment]
the botany seed vendor has 10 seeds that you can get via premium, each gold coin can be changed into 2 silver coins. the point is to make coins (and some extra) so that the botanists have the chance to get all the seeds each round if they really want to, without having to go through the hassle of mining or raiding places. the way it is right now, it'll add these coins to the loot spawner so they'll be randomly placed in maintenance each round. this means every round they'll have to look for the coins in different places so they still have to do work to unlock the premium strange seeds

**Comment by JohnGinnane** [comment]
Yeah but the smart ones only need one coin ;¬)

**Comment by Jujumatic** [comment]
remove coin string then reeee

**Comment by Pyko1** [comment]
why not just move the strange seeds in to bio processor so we dont make every other premium item common as fuck and give rnd free mats while on it?

**Comment by Jujumatic** [comment]
oh shit good idea. i can close this pr then

**Pull request closed without merging**